### ISSUE TITLE
ImportError: No module named tkinter     

### ISSUE THREAD

**Issue by nicolas6954** [GliderGeek/PySoar#143]
I am trying to run "python main_pysoar.py" and get the error "ImportError: No module named tkinter". Multiple attempts to get tkinter installed have failed, and following stackexchange and similar suggestions have not helped.  I have successfully run other python programs, such as igc_lib, but am no python expert.  I saw the screen capture on the page https://travis-ci.org/GliderGeek/PySoar/ and got the impression you were using Python 2.7 so I used virtualenv to try to set that environment instead of using Python 3.x which was installed with Ubuntu.  Also I tried the steps shown in the screen capture but still get the same error.  Any suggestions are welcome.  I'm running Ubuntu 18.04.  (I did try the standalone program under Windows and it worked very well.  I would like to get it working with USA contests which do not (currently) use SoaringSpot.)  Thanks in advance for any help.

P.S. I tried building the standalone version and got many errors, which I've attached in case they are useful, but I thought I'd start by focusing on running the .py file.
[warnmain_pysoar.txt](https://github.com/GliderGeek/PySoar/files/2717377/warnmain_pysoar.txt)

**Comment by GliderGeek** [comment]
@nicolas6954 : thanks for reporting your problem. i don't have the time to look at it now, but will let you know when i do. i the meantime, if you need core functionality used in pysoar, i suggest you use the package [opensoar](https://gitlab.com/gitlab-org/gitlab-ce/uploads/b60afe5c598e04c17d9ac6f6513d46cf/Dec-13-2018_11-50-26.gif). pysoar is merely a thin layer over this library adding UI and export funcionality

**Comment by nicolas6954** [comment]
Can you tell me which version of (a) Python and (b) tkinter you last used to successfully build the version of PySoar available for download from GitHub?  If you remember how you installed tkinter (presumably NOT with pip) that might be useful as well.  I suspect the tkinter problem is version related.   

I understand not having time to look into problems with free software.
Thanks in advance for any replies!

Nico

    On Wednesday, January 2, 2019, 4:41:53 AM EST, GliderGeek <notifications@github.com> wrote:  
 
 
@nicolas6954 : thanks for reporting your problem. i don't have the time to look at it now, but will let you know when i do. i the meantime, if you need core functionality used in pysoar, i suggest you use the package opensoar. pysoar is merely a thin layer over this library adding UI and export funcionality

**Comment by nicolas6954** [comment]
I think it would be a good idea if all python projects at least mentioned which version of python was last used for a successful build.  After learning more about python (e.g. https://www.hackerfactor.com/blog/index.php?/archives/825-8-Reasons-Python-Sucks.html) I find I am not the only one to have had version specific problems.

I finally got PySoar working under Ubuntu 18.04 with python 3.6 (included in my Ubuntu 18.04) as follows

- change the numpy version in requirements.txt from 1.12.0b1 to 1.14

I used the following command to install the proper tk

  sudo apt-get install python3-tk

Christian, thanks again for what looks writing what looks to be a very useful package!
